src/behaviors/deployer.rs

```diff
@@ -69,7 +69,16 @@ impl Behavior<()> for Deployer {
             pool,
         );
 
-        messager.send(To::All, serde_json::to_string(&deployment_data)?).await;
```

Comment:
Could just `?` here to save LOC as it is effectively the same thing

Reply:
yeah, prefer using this ? tbh, mistake on my path